### ISSUE TITLE
[HW] PRATEEK SHARMA

### ISSUE THREAD

**Issue by prateeksharmaaa** [datastaxdevs/workshop-graphql-netflix#190]
**Name:** PRATEEK SHARMA
**Email:** sharma.prateek362@gmail.com
**Linkedin Profile:** https://www.linkedin.com/in/prateek-sharma-70326b181

Attach the homework screenshots below:
-----------------------------------------

![Screenshot (46)](https://user-images.githubusercontent.com/52321558/124041386-6601bd80-da24-11eb-95fc-68f2ee8f6296.png)
![Screenshot (47)](https://user-images.githubusercontent.com/52321558/124041401-6a2ddb00-da24-11eb-9669-b4efef54b40f.png)
![Screenshot (48)](https://user-images.githubusercontent.com/52321558/124041415-6c903500-da24-11eb-80c3-d226e68ed1ae.png)
![Screenshot (49)](https://user-images.githubusercontent.com/52321558/124041419-6ef28f00-da24-11eb-90db-18fa48f3fa11.png)
![Screenshot (50)](https://user-images.githubusercontent.com/52321558/124041427-7154e900-da24-11eb-9cb4-7eb61a801304.png)

**Comment by EnushaManivannan** [comment]
Sir I couldn't figure out the messages I am receiving from datastax and
about the homework.
Can you please give instructions on how to do the homework and where and
how to learn from datastax.

**Comment by SonicDMG** [comment]
Of course. So, for the homework you need to add a video of your own. To do this, use the graphQL playground like we did to create the tables and insert some of the data. You can modify the queries we provide in the graphQL section https://github.com/datastaxdevs/workshop-graphql-netflix#4-insert-data-in-the-table-with-graphql to add your own video. You can grab an mp4 from a site like imgur.

**Comment by SonicDMG** [comment]
BTW, it all looks good, just need your custom added video, thank you. :)

**Comment by prateeksharmaaa** [comment]
I did add the video from giphy but that doesn't display, and there's no clip of passengers in imgur just images

**Comment by prateeksharmaaa** [comment]
> BTW, it all looks good, just need your custom added video, thank you. :)

Do I have to reupload my homework or just edit this?

**Comment by SonicDMG** [comment]
Just editing this is fine and I see your custom video. Thank you for following up. Here's your badge -> https://api.badgr.io/public/assertions/do6SMXMFQIKEAd854NH7Fg?identity__email=sharma.prateek362%40gmail.com
